magento/inventory
# MSI: Contains ui element for stock editor mask

### Description
adds website selection field to add/edit stock mask

### Manual testing scenarios

1. create a additional store
2. create a new stock and view die option selections and check if die main website and the additional store are there.

### Contribution checklist
